releases/_posts/2024-09-24-spark-release-3-5-3.md

```diff
@@ -22,7 +22,6 @@ Spark 3.5.3 is the third maintenance release containing security and correctness
 - [[SPARK-46037]](https://issues.apache.org/jira/browse/SPARK-46037) - When Left Join build Left, ShuffledHashJoinExec may result in incorrect results
 - [[SPARK-46632]](https://issues.apache.org/jira/browse/SPARK-46632) - EquivalentExpressions throw IllegalStateException
 - [[SPARK-48965]](https://issues.apache.org/jira/browse/SPARK-48965) - toJSON produces wrong values if DecimalType information is lost in as[Product]
-- [[SPARK-49030]](https://issues.apache.org/jira/browse/SPARK-49030) - Self join of a CTE seems non-deterministic
```

Comment:
does it mean these tickets miss specify the fixed version?

Reply:
Yes, they are not included (or even not resolved) in 3.5.3, so I also updated "Fix Version/s" to 3.5.4:

<img width="673" alt="Screenshot 2024-09-25 at 4 48 45 PM" src="https://github.com/user-attachments/assets/e5390339-a46e-49fd-8248-ad346d701ae6">